framework/util/file_path.cpp

```diff
@@ -435,6 +435,28 @@ void GetApplicationInfo(FileInfo& file_info)
     {
         GFXRECON_LOG_WARNING("Failed to retrieve the application executable name");
     }
+#elif defined(__ANDROID__) || defined(__linux__)
+    std::ifstream cmdlineFile("/proc/self/cmdline");
+
+    if (cmdlineFile.is_open())
+    {
+        std::string line;
+        std::getline(cmdlineFile, line);
+
+        if (!line.empty())
+        {
+            strcpy(file_info.AppName, line.c_str());
+            GetFileInfo(file_info, file_info.AppName);
+        }
+        else
+        {
+            GFXRECON_LOG_WARNING("Failed to open '/proc/self/cmdline' to get the application executable name");
```

Comment:
Please update this warning message to match the condition check (or maybe it goes away if you don't check `line.empty`)

Reply:
Done!